modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/product_options/ProductOptionCheckboxMultiple.js

```diff
@@ -114,7 +115,9 @@ const ProductOptionCheckboxMultiple = ({
 				skuOptionsAtomState
 			),
 			namespace,
```

Comment:
Note: we don't want to set namespace in the minicart. We only want to change minicart props.